include/base/getpot.h

```diff
@@ -202,6 +202,24 @@ class GetPot
   inline bool have_variable(const std::string& VarName) const;
 
   /**
+   * Check for a section name. When querying, the section_name
+   * MUST end in a forward slash. e.g.
+   * /Section1/
```

Comment:
Kind of fragile, no?  Can we test for a trailing forward slash and add it internally if absent?